packages/calcite-components/package.json

```diff
@@ -54,8 +54,12 @@
     "screenshot-tests:preview": "npm run util:prep-build-reqs && NODE_OPTIONS=--openssl-legacy-provider STORYBOOK_SCREENSHOT_LOCAL_BUILD=true storybook dev",
     "screenshot-tests:publish": "npm run screenshot-tests && storybook-to-ghpages --existing-output-dir=docs",
     "start": "npm run util:clean-js-files && concurrently --kill-others --raw \"trap 'npm run util:clean-js-files' INT TERM EXIT >/dev/null 2>&1 || true && tsc --project ./tsconfig-demos.json --watch\" \"npm run build:watch-dev\"",
-    "test": "vitest run",
-    "test:watch": "vitest",
+    "test": "concurrently \"npm:test:stable\" \"npm:test:experimental\"",
+    "test:stable": "STABLE_TESTS=true vitest run",
+    "test:experimental": "EXPERIMENTAL_TESTS=true vitest run --browser.headless",
+    "test:watch": "npm run test:watch:stable && npm run test:watch:experimental",
```

Comment:
Did you confirm `test:watch` works? I don't think `test:experimental` will run because `test:stable` won't exit. You may need to use `concurrently` like the `test` script does.

Reply:
Great catch. Will fix!

Reply:
Used `concurrently` + passthrough options. One thing to note is that interactive watch mode won't work as before since both test runners might conflict when handling input. I've updated the contributing doc with this.